src/main/java/org/openstreetmap/atlas/checks/atlas/predicates/TagPredicates.java

```diff
@@ -53,6 +53,26 @@ private TagPredicates()
     public static final Predicate<AtlasObject> IS_HIGHWAY_AREA = object -> Validators.hasValuesFor(
             object, HighwayTag.class) && Validators.isOfType(object, AreaTag.class, AreaTag.YES);
 
+    /**
+     * Tests if the {@link AtlasObject} has a highway tag that do not contain TERTIARY_LINK,
+     * SECONDARY_LINK, PRIMARY_LINK, TRUNK_LINK, or MOTORWAY_LINK
+     */
+    public static final Predicate<AtlasObject> IS_HIGHWAY_NOT_LINK_TYPE = object -> Validators
+            .hasValuesFor(object, HighwayTag.class)
+            && Validators.isNotOfType(object, HighwayTag.class, HighwayTag.TERTIARY_LINK,
+                    HighwayTag.SECONDARY_LINK, HighwayTag.PRIMARY_LINK, HighwayTag.TRUNK_LINK,
+                    HighwayTag.MOTORWAY_LINK);
+
+    /**
+     * Tests if the {@link AtlasObject} has a highway tag that do contain TERTIARY, SECONDARY,
+     * PRIMARY, TRUNK, or MOTORWAY
+     */
+    public static final Predicate<AtlasObject> VALID_HIGHWAY_TAG = object -> Validators
```

Comment:
I feel like this is specific to this check and should go within the check class than in the TagPredicates. Valid highways for this check may not be the same for other checks.

Reply:
Makes sense. Will move it to current file.